src/module/system/damage/roll.ts

```diff
@@ -249,6 +259,7 @@ class DamageRoll extends AbstractDamageRoll {
             total: isPrivate ? "?" : Math.floor((this.total! * 100) / 100),
             increasedFrom: this.options.increasedFrom,
             splashOnly: !!this.options.splashOnly,
+            healingOnly: !this.kinds.has("damage"),
```

Comment:
Might need to test if it has healing as well, in case there are no instances somehow?

Reply:
Something would have gone so catastrophically wrong in that case that healing/damage would be the least of the problems.

Reply:
There's even a guard against that earlier, huh.